### PR TITLE
get a list of github repos and display on screen

### PR DESCRIPTION
**Pull request by JacobWashburn**
# Description
created a modal that searches github for repos and allows you to add repos to a list. you can then click save and the selected repos will be displayed. static grades and links for now.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by jiangeyre — APPROVED** [review]
Pulled the branch's commits locally. Tested the code - add, remove, search - all work successfully! Great code. 10/10 feature.

File: src/components/Project/index.jsx [hunk 1]
```diff
@@ -1,97 +1,99 @@
...
+            {user == true ? (
+              <NoteEditor
+                executeQuery={executeQuery}
+                user={data.me}
+                projectId={id}
+                projectManagers={data.project.projectManagers}
+              />
+            ) : null}
+
+            {/*<NotesFeed projectId={id} privateBol={user} />*/}
```
Comment (lawsonarichard) [inline src/components/Project/index.jsx]: Comment this back in. You're removing functionality.

**Review by lawsonarichard — CHANGES_REQUESTED** [review]
See comments!

File: src/components/Project/index.jsx (hunk 1 above)
Comment (JacobWashburn) [reply src/components/Project/index.jsx]: Done.

**Review by lawsonarichard — APPROVED** [review]

**Pull request merged**